src/ImageSharp.Web.Providers.AWS/AmazonS3ClientFactory.cs

```diff
@@ -33,7 +32,7 @@ public static AmazonS3Client CreateClient(IAWSS3BucketClientOptions options)
         else if (!string.IsNullOrWhiteSpace(options.AccessKey))
         {
             // AccessSecret can be empty.
-            Guard.NotNullOrWhiteSpace(options.Region, nameof(options.Region));
+            Guard.NotNullOrWhiteSpace(options.Region!, nameof(options.Region));
```

Comment:
This reads like we need to update `Guard`? We cannot assert a string is non-null when passing it as a parameter for a null check.